air_link/package.py

```diff
@@ -19,6 +19,18 @@ def sorted_nicely(paths: List[Path]) -> List[Path]:
     return sorted(paths, key=lambda path: [int(c) if c.isdigit() else c for c in re.split('([0-9]+)', path.stem)])
 
 
+def write_env(target_folder: Path) -> None:
+    Path(target_folder / '.env').write_text(app.storage.general.get('env', ''))
+
+
+def read_env(target_folder: Path) -> None:
+    app.storage.general['env'] = Path(target_folder / '.env').read_text()
```

Comment:
When there is no `.env` file already this results in an error on startup.

Reply:
fixed with https://github.com/zauberzeug/air-link/pull/14/commits/6381dcef4f14a271ff82b3a4143b2bc79fbb0132